frontend/packages/kubevirt-plugin/src/utils/url.ts

```diff
@@ -45,3 +46,22 @@ export const resolveURL = ({ urlObj, maxHostnameParts, maxPathnameParts }) =>
   urlObj.origin === 'null'
     ? urlObj.href
     : `${resolveOrigin(urlObj, maxHostnameParts)}${resolvePathname(urlObj, maxPathnameParts)}`;
+
+export const getVMWizardCreateLink = (
+  namespace: string,
+  itemName: VMWizardName = VMWizardName.WIZARD,
+  mode: VMWizardMode = VMWizardMode.TEMPLATE,
+  template?: string,
+) => {
+  // Overide mode if name is import.
+  const wizardMode = itemName === VMWizardName.IMPORT ? VMWizardMode.IMPORT : mode;
+  const type = itemName === VMWizardName.YAML ? '~new' : '~new-wizard';
+
+  const params = new URLSearchParams();
+  params.append('mode', wizardMode);
```

Comment:
setting the vm mode isn't necessary as it is the default fallback

Reply:
but it doesn't heart ... hot setting it requires an `if ... ` that we don't really need.

Reply:
it creates urls with parameters - which the user can copy as a shortcut for the vm creation.

And it is a bit nicer to have simpler urls and show the user that they exist

Reply:
ok :-)